recipe-client-addon/lib/ClientEnvironment.jsm

```diff
@@ -174,6 +179,15 @@ this.ClientEnvironment = {
       return Preferences.get("privacy.donottrackheader.enabled", false);
     });
 
+    XPCOMUtils.defineLazyGetter(environment, "experiments", async () => {
+      const experiments = await PreferenceExperiments.getAll();
+      return {
+        all: experiments.map(e => e.name),
+        active: experiments.filter(e => !e.expired).map(e => e.name),
+        expired: experiments.filter(e => e.expired).map(e => e.name),
```

Comment:
Technically, it'd be more efficient to just have 1 for loop and add the names to the right arrays. Meh. Up to you whether you do something about this, I guess.